.travis.yml

```diff
@@ -9,24 +9,15 @@ cache:
     - .pants.d
     - $HOME/.cache/pants
 
-matrix:
-  include:
-    - python: "2.7"
-      env:
-        - TOX_ENV=py27
-    - python: "3.4"
-      env:
-        - TOX_ENV=py34
-    - python: "3.5"
-      env:
-        - TOX_ENV=py35
-    - python: "3.6"
-      env:
-        - TOX_ENV=py36
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"
```

Comment:
Having problems over at django-wiki with this, too... but this does the trick: http://tox-travis.readthedocs.io/en/stable/